README.md

```diff
@@ -1,7 +1,124 @@
-# Dialogue
+<p align="right">
+<a href="https://autorelease.general.dmz.palantir.tech/palantir/dialogue"><img src="https://img.shields.io/badge/Perform%20an-Autorelease-success.svg" alt="Autorelease"></a>
+</p>
 
-Dialogue is an experimental client-side framework for HTTP-based RPC. The API is influenced by gRPC's Java library.
-Dialogue was designed as a codegen target for [Conjure](https://palantir.github.io/conjure).
+# Dialogue ![Bintray](https://img.shields.io/bintray/v/palantir/releases/dialogue.svg) [![License](https://img.shields.io/badge/License-Apache%202.0-lightgrey.svg)](https://opensource.org/licenses/Apache-2.0)
+
+_Dialogue is a client-side library for HTTP-based RPC, designed to work well with [Conjure](https://palantir.github.io/conjure)-defined APIs._
+
+## Features
+
+- **ConcurrencyLimiters**: additive increase multiplicative decrease (AIMD) concurrency limiters ensure bursty traffic doesn't overload upstream servers.
+- **Client-side node selection**: by making load balancing decisions in the client, Dialogue avoids the necessity for an L7 proxy (and its associated latency penalty).
+- **Queue**: in the case where all nodes are limited (e.g. during a spike in traffic), requests are added to a FIFO queue and processed as soon as the one of the ConcurrencyLimiters has capacity.
+- **Retries**: requests are retried a constant number of times, unless they were streamed.
```

Comment:
nit, no change required though: on my list to support repeatable streaming requests
```suggestion
- **Retries**: requests are retried a constant number of times, if possible.
```